### PR TITLE
Add documentation for command-line interface

### PR DESCRIPTION
**Pull request by renaissanceGeek**
You only need to review the markdown files.  You can see them as formatted by hugo by port-forwarding from coby:

`ssh -4 -L 1313:localhost:1313 coby`

then browse to http://localhost:1313/sdk-cubic/cli/

Two broader questions:
1. Should these documents be in the cubic-cli repo instead?  It feels like they should be, but it also seems like it would be very useful to have all the docs together in https://cobaltspeech.github.io.  Can we have the same hugo navigation for docs in different repos?
2. Writing up this documentation makes it even more clear that the output formats could use some thought and cleanup.

### CODE REVIEW

File: docs-src/content/cli/Transcribe.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+There are two audio files in the testdata folder.
+They are US English, recorded at 16kHz.
+The filenames contain the intended transcription.
+
+As a quick start, these commands are provided as examples of how to use the
+binary.  They should be run from the root directory of `cubic-cli`.
+
+Note: These commands assume that the your instance of cubic server is available
+at `localhost:2727`.
```
[the remark this answers is not in the document]
Comment (shahruk10) [reply docs-src/content/cli/Transcribe.md]: Move the bit about using demo page from below to here ?

File: docs-src/content/cli/Formats.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+    * **confidence**: Number between 0 and 1 representing the probability of this result vs the other alternatives
+    * **transcript**: Full result (up to the endpoint)
+    * **words**: Array of the individual confidence, duration, and start time for each word in the transcript.  Can be used to provide finer alignment to the audio.
+
+### Example
+```bash
+bin/cubic-cli -s localhost:2727 transcribe -m 1 -f timeline --stereo --insecure 2-channel.wav
+```
+
+```json
```
[the remark this answers is not in the document]
Comment (renaissanceGeek) [reply docs-src/content/cli/Formats.md]: Nice!

File: docs-src/content/cli/Formats.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+  }
+]
+```
+## Json and Json-pretty
+
+Both `json` and `json-pretty` return the same data.
+The only difference is that `json-pretty` includes line breaks and indentation to make it more human-readable.
+Both formats return all alternatives recognized for each result. 
+Word-level information is not included and the results are the order cubic processed them, so results from 
+different channels are not guaranteed to be ordered according to their timestamps.
```
[the remark this answers is not in the document]
Comment (renaissanceGeek) [reply docs-src/content/cli/Formats.md]: I dislike the default json encoding for durations, so I'm hoping we can change how it's output, but I guess until/unless we do, we should at least explain it.

**Comment by renaissanceGeek** [comment]
> I think we should move this documentation to the cubic-cli repo, rather than host it here:
> 
> 1. we should include a section here suggesting that we have a client and point to the repository there.
> 2. I think it's OK if that documentation is "independent" and not connected to the nav structure here.
> 3. Having it separate would make it easier for the documentation and the code be in sync and versioned correctly.
> 
> Can we please move it there and do the actual PR review there?

OK, I'll move the documentation--I did adopt Shahruk's suggestions here first, though, just to close the loop on them so they didn't get lost in the confusion of setting up Hugo docs in a different repo.

**Comment by renaissanceGeek** [comment]
Moved these docs to the cubic-cli repo: https://github.com/cobaltspeech/cubic-cli/pull/22

**Pull request closed without merging**